docs/documentation/make-first-prototype/let-user-change-answers.md

```diff
@@ -0,0 +1,43 @@
+# Let the user change their answers
+
+## Make the ‘Change’ links work
+
+Make the **Change** links on the ‘Confirmation’ page work by adding the right links.
+
+1. In the the <a> tag under `{{ data['how-many-balls'] }}`, change the href attribute from `#` to ‘/juggling-balls’
```

Comment:
Anchor tag needs to be wrapped in backticks – it's currently not showing up because it's being interpreted as HTML.

```suggestion
1. In the the `<a>` tag under `{{ data['how-many-balls'] }}`, change the href attribute from `#` to ‘/juggling-balls’
```

(and on the following line)

Reply:
Thanks for spotting - all the markdown issues you noticed should be fixed in https://github.com/alphagov/govuk-prototype-kit/pull/805/commits/871d36b80c663194857e7a0708d31a7b6e213ff2